@staticmethod
def exception():
    return PythonTestResult('EXCEPTION')
